app/code/Magento/Downloadable/view/frontend/templates/catalog/product/links.phtml

```diff
@@ -49,7 +49,8 @@
                     <input type="checkbox"
                            data-notchecked="<?= $block->escapeHtmlAttr(__('Select all')) ?>"
                            data-checked="<?= $block->escapeHtmlAttr(__('Unselect all')) ?>"
-                           id="links_all" />
+                           id="links_all"
+                           name="links_all" />
```

Comment:
Adding name to checkbox will send this value to the server with form submitting. Is it possible to adjust JS selector?

Reply:
done